app/MMB/src/scenes/tripServices/insurance/paymentScene/CardImage.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+
+import * as React from 'react';
+import { Image } from 'react-native';
+
+type CardType =
+  | 'MASTERCARD'
+  | 'VISA'
+  | 'MAESTRO'
+  | 'AMERICAN_EXPRESS'
+  | 'DISCOVER'
+  | 'DINERS_CLUB'
+  | 'MIR'
+  | 'UNKNOWN';
+
+type Props = {|
+  +cardType: CardType,
+|};
+
+const CardImage = ({ cardType }: Props) => {
+  let source;
+  switch (cardType) {
+    case 'MASTERCARD': {
+      source = require('./images/MasterCardLogo.imageset/mastercard.png');
+      break;
```

Comment:
Does this really work?

https://github.com/facebook/react-native/issues/6391#issuecomment-194568185

Reply:
Well, the image was changing when I tried different card numbers